packages/conf-qt/conf-qt.5.2.1/opam

```diff
@@ -12,12 +12,15 @@ dev-repo:     "https://github.com/Kakadu/lablqml.git"
 build: [
     ["sh" "-exc" "echo \"Your Qt version is `PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH qmake -query QT_VERSION`\"" ] 
     ["sh" "-exc" "min=\"5.2.1\" cur=`PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH qmake -query QT_VERSION`; res=`printf \"$min\\n$cur\\n\" | sort | head -n 1`; [ \"$res\" = \"$min\" ]"
-    ] 
+    ]
 ]
 
-available: [ os != "darwin" ]
+post-messages: [
+  "It's recommended to set PKG_CONFIG_PATH to /usr/local/opt/qt/lib/pkgconfig to enable pkg-config" { os = "darwin" }
+]
 
 depexts: [
+  [ ["osx" "homebrew"] ["qt5"] ]
```

Comment:
Yeah, I heard about it. But folks say that even without this command executed automatically this pull request will be a progress.

https://github.com/Kakadu/lablqml/issues/21#issuecomment-319489574